### ISSUE TITLE
Determine what the minimum donation amount is and add validations

### ISSUE THREAD

**Issue by jagthedrummer** [techlahoma/techlahoma_donations#60]
It probably doesn't make good sense to allow people to donate $1 since fully 1/3rd of that will be eaten by Stripe fees. We need to determine what a reasonable minimum donation level is and add validations that prevent someone from getting through "checkout" with less.

**Comment by groovecoder** [comment]
Wait, we're not adding stripe fees to the charge? We absolutely do this for codesy where we expect lots of amounts to be in the ~$5 range. I.e., if you add $1 to a bug, we charge $1.32

So, a $32 donation would be 32 + .29 + .025*32 = 33.09.
$64 = $65.89
$128 = $131.49
etc.

**Comment by jagthedrummer** [comment]
No, we don't tack on the Stripe fees on top of what people indicate they want to contribute. If you tell us you want to give us $32 we take the $32 and the fact that we have to pay a fee is our problem. I'm not entirely against changing that, but it's pretty non-standard and not generally what people expect.

**Comment by energydev** [comment]
For charitable donations via debit or credit card I've seen, the amount donated, "X dollars" was always the amount charged.  For profit purchases are the ones where it is sometimes standard to add on the credit card processing fees.

**Comment by energydev** [comment]
I recommend, for single donations, a minimum $32 donation for memberships. If the "other" option is selected, we could put wording "Min $32". Given that memberships may receive discounts from places like 36 Degrees and that we're planning to provide year end gifts, $32 seems like a reasonable minimum.

**Comment by groovecoder** [comment]
We need @dustincurzon to tell us what the minimum 1-time donation amount should be to qualify for a 36°N discount. Last year it was $128/year minimum to receive $50/mo. discount. I.e., spending $128 to save up to $600.

**Comment by dustincurzon** [comment]
I like the $128 minimum - not really for 36N's sake but I'm all for incentivizing people to give more to Techlahoma.






On Wed, Mar 15, 2017 at 5:27 PM -0500, "luke crouch" <notifications@github.com> wrote:












We need @dustincurzon to tell us what the minimum 1-time donation amount should be to qualify for a 36°N discount. Last year it was $128/year minimum to receive $50/mo. discount. I.e., spending $128 to save up to $600.

**Comment by jagthedrummer** [comment]
@groovecoder @dustincurzon I've heard from _numerous_ people that it's too high of a minimum and that it prevents them from joining as a member all together. So I think we need some lower options. $64 at the very least, if not $32.

That doesn't necessarily mean that 36°N needs to offer the discount at those levels. I don't see a problem with saying "Techlahoma members who donate $XXX or more qualify for a YY% discount."

**Comment by jagthedrummer** [comment]
In fact, after thinking about it, I'm inclined to make $16 the minimum, just to potentially open up to a wider audience.

**Comment by dustincurzon** [comment]
@jagthedrummer I agree on the lower minimum for donations. Anyone who wants to give money, let's take it. Keeping perks like 36°N membership at higher levels incentives the upsell.

**Comment by groovecoder** [comment]
Yeah, sorry I meant just to reserve the 36°N discount for higher level donations.

**Comment by energydev** [comment]
I like the idea of a $16 single donation minimum, with the 36°N discount reserved for higher level donations. What about the minimum for monthly donations? Maybe $8/month minimum? That's $96/year. It's about $90/year after the stripe fees are considered.

**Comment by energydev** [comment]
If there is agreement from 36°N on a discount for $128 donations and above, I'd encourage us to add that wording to the website before we go live with the membership campaign site. Also, would we be good with $16/month also getting the discount. That's $192 over 12 months or $144 over 9 months. The next lowest per month is $8/month, which is $96 across 12 months.

**Comment by dustincurzon** [comment]
I agree with the $16/month level for 36N

**Comment by groovecoder** [comment]
$16/mo. to *save* $50/mo. - what a deal!! Since we've got an SS46/36°N exchange program going on, we should make sure SS46 is okay with those levels too?

**Comment by jagthedrummer** [comment]
@groovecoder @dustincurzon thanks for the conversation on this. We should definitely verify with SS46 if there's an expectation that both spaces will do the same thing. Last night I was chatting with @DevinClark and he brought up a good point. Last year when we were setting up the drive, Mike Slack, our fundraising consultant, said that we can't directly offer discounts as part of the benefits for membership unless we're going to do the math/accounting to deduct that from their tax deductible donation amount, since it would be a "service" that they receive. I'm going to confirm with him exactly what the details are on that, but just wanted you guys to know. It _might_ be one of those things where Techlahoma can't push it as a benefit, but that 36DN and SS46 could say "we like Techlahoma, and for their members we offer a discount". Kind of like how places offer student discounts, or senior discounts, or fire-fighter discounts, or whatever. I'll let you know what he says.

**Comment by jagthedrummer** [comment]
Ok, I just pushed this commit https://github.com/techlahoma/techlahoma_donations/commit/35fd39ea845be0bc581eeb7ffe374d035e79dfe4, which:
* adds an $8/moth recurring level
* adds $16 to the recommended donation levels
* validates that the donation level select has been set to something other than the prompt (on the one-off form)
* validates that if someone chooses 'Other' in the one-off donation form that the amount they enter must be $10 or more

I'll leave this open for now to remind us that we need to wrap up the conversation about offering discounts at SS46 and 36DN.

**Comment by energydev** [comment]
Yes, we definitely need to get clarity on the discounts, especially since those not using the discounts wouldn't want that to impact their tax deductions.
